### ISSUE TITLE
gcc 5gcc-v

### ISSUE THREAD

**Issue by reijin90** [fireice-uk/xmr-stak-cpu#339]

**Comment by psychocrypt** [comment]
Have you changed something in the cmake file?
Please run `cmake . -L` within the build folder.

**Comment by reijin90** [comment]
-- Configuring done
-- Generating done
-- Build files have been written to: /home/xmr/xmr-stak-cpu
-- Cache values
CMAKE_BUILD_TYPE:STRING=Release
CMAKE_INSTALL_PREFIX:PATH=/home/xmr/xmr-stak-cpu
CMAKE_LINK_STATIC:BOOL=OFF
HWLOC:FILEPATH=/usr/lib/x86_64-linux-gnu/libhwloc.so
HWLOC_ENABLE:BOOL=ON
HWLOC_INCLUDE_DIR:PATH=/usr/include
MHTD:FILEPATH=/usr/lib/x86_64-linux-gnu/libmicrohttpd.so
MICROHTTPD_ENABLE:BOOL=ON
MTHD_INCLUDE_DIR:PATH=/usr/include
OpenSSL_ENABLE:BOOL=ON

**Comment by fireice-uk** [comment]
@psychocrypt 

This is the tell tale error :

> /home/xmr/xmr-stak-cpu/jpsock.h:70:9: note: C++11 'constexpr' only available with -std=c++11 or -std=gnu++11

For some reason @reijin90 is building without that flag.

**Comment by psychocrypt** [comment]
Yes thats the problem but we compile the full project with c++11 expect all
C files in crypto.

Am 22.09.2017 3:41 Nachm. schrieb "fireice-uk" <notifications@github.com>:

**Comment by Ogy** [comment]
do anybody have a solution for this?

**Comment by reijin90** [comment]
Sadly I didnt manage to find any solution for this :(

**Comment by reijin90** [comment]
Ok, I redownloaded everything after I made sure the correct gcc version was installed and then it compiled. But: I didn't follow the Linux instructions, and instead of "make install" I only issued "make" this time and it worked! :)

**Comment by cooler0918** [comment]
I have a similar issue trying to run 'make'

I had to specify the compiler manually since it would not find it on it's own.

`root@XMR_Miner:~/xmr-stak # make
[ 24%] Built target xmr-stak-c
[ 28%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/version.cpp.o
In file included from /root/xmr-stak/xmrstak/version.cpp:1:0:
/root/xmr-stak/xmrstak/version.hpp: In function 'std::__cxx11::string get_version_str()':
/root/xmr-stak/xmrstak/version.hpp:13:33: error: 'to_string' is not a member of 'std'
  return std::string(ver_long) + std::to_string(uint32_t(fDevDonationLevel * 1000)) ;
                                 ^
*** [CMakeFiles/xmr-stak-backend.dir/xmrstak/version.cpp.o] Error code 1

Stop in /root/xmr-stak.
*** [CMakeFiles/xmr-stak-backend.dir/all] Error code 1

Stop in /root/xmr-stak.
*** [all] Error code 1

Stop in /root/xmr-stak.`

`root@XMR_Miner:~/xmr-stak # cmake . -L
-- Set miner currency to 'monero' and 'aeon'
-- Configuring done
-- Generating done
-- Build files have been written to: /root/xmr-stak
-- Cache values
CMAKE_BUILD_TYPE:STRING=Release
CMAKE_INSTALL_PREFIX:PATH=/root/xmr-stak
CMAKE_LINK_STATIC:BOOL=OFF
CPU_ENABLE:BOOL=ON
CUDA_ENABLE:BOOL=OFF
HWLOC:FILEPATH=/usr/local/lib/libhwloc.so
HWLOC_ENABLE:BOOL=ON
HWLOC_INCLUDE_DIR:PATH=/usr/local/include
MHTD:FILEPATH=/usr/local/lib/libmicrohttpd.so
MICROHTTPD_ENABLE:BOOL=ON
MTHD_INCLUDE_DIR:PATH=/usr/local/include
OpenCL_ENABLE:BOOL=OFF
OpenSSL_ENABLE:BOOL=ON
WIN_UAC:BOOL=ON
XMR-STAK_COMPILE:STRING=native
XMR-STAK_CURRENCY:STRING=all`
